pallets/avn/src/lib.rs

```diff
@@ -393,13 +393,21 @@ impl<T: Config> Pallet<T> {
     ) -> bool {
         // verify that the incoming (unverified) pubkey is actually a validator
         if !Self::is_validator(&validator.account_id) {
+            warn!("Signature validation failed, account {:?}, is not validator", validator);
             return false
         }
 
         // check signature (this is expensive so we do it last).
         let signature_valid =
             data.using_encoded(|encoded_data| validator.key.verify(&encoded_data, &signature));
 
+        debug!(
+            "🪲 Validating signature: [ data {:?} - account {:?} - signature {:?} ] Result: {}",
```

Comment:
is the emoji a `bug`? :smile:

Reply:
cause it is **de**-bug level of info :tada: